include/mitsuba/core/properties.h

```diff
@@ -17,7 +17,7 @@ template <typename T, typename = void> struct prop_map { using type = void; };
 template <typename T> using prop_map_t = typename prop_map<std::decay_t<T>>::type;
 template <typename T> struct base_type;
 template <typename T> struct is_transform_3: std::false_type { };
-template <typename T> struct is_transform_3<Transform<Point<T, 3>>> : std::true_type { };
+template <typename T> struct is_transform_3<AffineTransform<Point<T, 3>>> : std::true_type { };
```

Comment:
According to the PR description, shouldn't this be `ProjectiveTransform`?
Properties should use the most general / powerful transform type I think?

Reply:
We never pass projective transformations to plugins